include/swift/AST/ModuleDependencies.h

```diff
@@ -37,7 +37,8 @@ class Identifier;
 /// Which kind of module dependencies we are looking for.
 enum class ModuleDependenciesKind : int8_t {
   FirstKind,
-  SwiftTextual = FirstKind,
+  SwiftInterface = FirstKind,
+  SwiftSource,
```

Comment:
Minor nitpick: new cases are generally preferred to be appended at the last one to preserve stability of the existing cases.